src/keyboard.c

```diff
@@ -47,7 +47,7 @@ static int keyboard_scan() {
     int code;
     int ack;
 
-    code = inb(KEYBOARD_PORT_A);
+    code = inb(PS2_DATA_PORT);
     iowait();
     ack = inb(KEYBOARD_PORT_B);
```

Comment:
Thain used it. not a specifically PS2 port. From OS Dev Wiki: 

Historical note : The PC-XT PPI had used port 0x61 to reset the keyboard interrupt request signal (among other unrelated functions). Port 0x61 has no keyboard related functions on AT and PS/2 compatibles.